### PR TITLE
Support for setting the environment

### PR DESCRIPTION
**Pull request by boldbart**
Hi Just Better, we added a configuration setting for sending the environment variable. In our case this was necessary, because of acceptance environments that are running in production mode. See https://docs.sentry.io/enriching-error-data/environments/?platform=javascript

### CODE REVIEW

File: Model/SentryLog.php [hunk 1]
```diff
@@ -62,6 +62,11 @@ public function send($message, $logLevel, Monolog $monolog, $context = [])
 
         if ($logLevel >= (int) $config['log_level']) {
             $client = (new Raven_Client($config['domain'] ?? null));
+
+            if(!is_null($config['environment'])) {
```
Comment (royduin) [inline Model/SentryLog.php]: This isn't PSR compliant, don't know if anything is but it should. @rakibabu maybe integrate https://styleci.io?

```suggestion
            if (!is_null($config['environment'])) {
```

**Comment by rakibabu** [comment]
@boldbart 

Thanks for contributing! Can you resolve the merge conflicts with version 0.5.0 then I'll merge this PR.

**Comment by royduin** [comment]
Thank you! @rakibabu do you create a new release?

**Comment by rakibabu** [comment]
@royduin & @boldbart sorry for the delay, but this PR is in released in 0.5.1

**Pull request merged**